mlflow/projects/databricks.py

```diff
@@ -74,18 +66,24 @@ def _upload_to_dbfs(self, src_path, dbfs_fuse_uri):
         eprint("=== Uploading project to DBFS path %s ===" % dbfs_fuse_uri)
         http_endpoint = dbfs_fuse_uri
         with open(src_path, 'rb') as f:
-            response = self._databricks_api_request(endpoint=http_endpoint, method='POST', data=f)
-            _check_response_status_code(response)
+            self._databricks_api_request(endpoint=http_endpoint, method='POST', data=f)
 
     def _dbfs_path_exists(self, dbfs_path):
         """
         Return True if the passed-in path exists in DBFS for the workspace corresponding to the
         default Databricks CLI profile. The path is expected to be a relative path to the DBFS root
         directory, e.g. 'path/to/file'.
         """
-        response = self._databricks_api_request(
-            endpoint="/api/2.0/dbfs/get-status", method="GET", json={"path": "/%s" % dbfs_path})
-        json_response_obj = json.loads(response.text)
+        host_creds = databricks_utils.get_databricks_host_creds(self.databricks_profile)
+        response = rest_utils.http_request(
```

Comment:
Here we still use the old `http_request` API since non-200 response statuses can indicate that no file exists at the specified DBFS path